tests/test_onnx_export.py

```diff
@@ -610,12 +615,12 @@ def forward(self, inp):
         if precision in (torch.bfloat16, ):
             return
         if not use_fp8:
-            validate_result(fname, inp, model, atol=1e-3)
+            validate_result(fname, inp, model, atol=5e-4)
         else:
-            validate_result(fname, inp, model, atol=1e-3, is_fp8=use_fp8)
+            validate_result(fname, inp, model, atol=5e-4, is_fp8=use_fp8)
 
 
-@pytest.mark.parametrize("scale_factor", [112])
+@pytest.mark.parametrize("scale_factor", [[448, 448]])
```

Comment:
```suggestion
@pytest.mark.parametrize("scale_factors", [[448, 448]])
```

Made it explicitly plural